gadgets/profile_tcprtt/program.bpf.c

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2021 Wenbo Zhang
+// Copyright (c) 2023 The Inspektor Gadget authors
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_endian.h>
+#include <gadget/bits.bpf.h>
+#include <gadget/maps.bpf.h>
+#include <gadget/types.h>
+#include <gadget/macros.h>
+
+#ifndef PROFILER_MAX_SLOTS
+#define PROFILER_MAX_SLOTS 27
+#endif /* !PROFILER_MAX_SLOTS */
+
+#define IPV6_LEN 16
+
+/* Taken from kernel include/linux/socket.h. */
+#define AF_INET 2 /* IP version 4			*/
+#define AF_INET6 10 /* IP version 6			*/
+
+const volatile __u16 targ_sport = 0;
+const volatile __u16 targ_dport = 0;
+const volatile __u32 targ_saddr = 0;
+const volatile __u32 targ_daddr = 0;
+const volatile __u8 targ_saddr_v6[IPV6_LEN] = {};
+const volatile __u8 targ_daddr_v6[IPV6_LEN] = {};
+const volatile bool targ_ms = false;
+
+GADGET_PARAM(targ_sport);
+GADGET_PARAM(targ_dport);
+GADGET_PARAM(targ_saddr);
+GADGET_PARAM(targ_daddr);
```

Comment:
Having to specify the address using decimal notation is not very friendly. I'd suggest to remove these until we support it.